stylesheets/components/drawers/_drawer.styl

```diff
@@ -93,6 +93,15 @@
     ul:last-child
       margin-bottom: 0
 
+    @media $media-small-max
+      .content
+        padding-left: 0 !important
```

Comment:
Is there any way to either make this more specific or even better whatever it's overriding less specific to avoid using `!important`?